patroni/postgresql.py

```diff
@@ -59,7 +60,8 @@ def __init__(self, config):
         self.callback = config.get('callbacks') or {}
         self.use_slots = config.get('use_slots', True)
         self._schedule_load_slots = self.use_slots
-        self._postgresql_conf = os.path.join(self._data_dir, 'postgresql.conf')
+        self._postgresql_conf = os.path.join(self._data_dir,
+                                             config.get('config', 'postgresql.conf'))
```

Comment:
It will work. If the ``config_base_name`` was redefined in the config file Patroni will write not into postgresql.conf, but into pipelinedb.conf